onnx2torch/node_converters/eye.py

```diff
@@ -0,0 +1,50 @@
+__all__ = [
+    'OnnxEyeLike',
+]
+
+import torch
+from torch import nn
+
+from onnx2torch.node_converters.registry import add_converter
+from onnx2torch.onnx_graph import OnnxGraph
+from onnx2torch.onnx_node import OnnxNode
+from onnx2torch.utils.common import OnnxToTorchModule
+from onnx2torch.utils.common import OperationConverterResult
+from onnx2torch.utils.common import onnx_mapping_from_node
+
+
+class OnnxEyeLike(nn.Module, OnnxToTorchModule):  # pylint: disable=missing-docstring
+    def __init__(self, eyelike_k: int, dtype: torch.dtype = None):
+        super().__init__()
+        self.dtype = dtype
+        self.eyelike_k = eyelike_k
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:  # pylint: disable=missing-function-docstring
+        if len(x.shape) != 2:
+            raise ValueError('OnnxEyeLike only support 2-D tensor')
+
+        input_value_n = x.size(dim=0)
+        input_value_m = x.size(dim=1)
+        if self.eyelike_k > input_value_n:
+            raise ValueError(
+                f'Error EyeLike Attribute k value, the k value is {self.eyelike_k},'
+                'but x shape is {(input_value_n, input_value_m)}'
+            )
+
+        if self.eyelike_k == 0:
+            return torch.eye(input_value_n, input_value_m, dtype=self.dtype)
+
+        k_tensor = torch.zeros(input_value_n, self.eyelike_k, dtype=self.dtype)
+        eye_tensor = torch.eye(input_value_n, input_value_m - self.eyelike_k, dtype=self.dtype)
```

Comment:
Could you please use keyword arguments when number of args > 2?

Reply:
Sorry, i don't understand what's ` number of args > 2`.

Reply:
@JohnMasoner 

When the number of arguments you pass to a function is more than two, it is useful to pass them explicitly by name (as keyword arguments) to ensure readability, in this case:

```python
eye_tensor = torch.eye(n=input_value_n, m=input_value_m - self.eyelike_k, dtype=self.dtype)
```

For `torch.zeros` it is not possible, because [zeros](https://pytorch.org/docs/stable/generated/torch.zeros.html) only supports positional args for `size`.


_p.s. Could you update our OP table for EyeLike?_

Reply:
@senysenyseny16 

Thank you for your explanation. It corrected my previous bad coding habits.